### ISSUE TITLE
Argument which only has empty string or line break causes broken layout.

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#54]
```
$ tmux -V; xpanes -V
tmux 2.4
xpanes 2.0.2
```

This command causes broken layout **sometimes**.

```
$ xpanes '' 2 '\n' 3
```

Like this

![screen shot 2017-05-14 at 6 29 26 pm](https://cloud.githubusercontent.com/assets/3874767/26032638/78e64c96-38d3-11e7-98e1-e07d2575a9d7.png)

dump
```
...
+ tmux -S /root/.cache/xpanes/socket send-keys -t xpanes-11535:tmp-11535 '/usr/bin/xpanes '\''-c'\'' '\''echo {} '\''  '\''--'\'' '\'''\'' '\''2'\'' '\''\n'\'' '\''3'\'' ' C-m
+ tmux -S /root/.cache/xpanes/socket send-keys -t xpanes-11535:tmp-11535 exit C-m
+ '[' 1 -eq 1 ']'
+ tmux -S /root/.cache/xpanes/socket attach-session -t xpanes-11535
[exited]
+ '[' 0 -ne 0 ']'
+ xpns_restore_allow_rename on /root/.cache/xpanes/socket
+ local _default_allow_rename=on
+ local _session=/root/.cache/xpanes/socket
+ [[ on == \o\n ]]
+ '[' -z /root/.cache/xpanes/socket ']'
+ tmux -S /root/.cache/xpanes/socket set-window-option -g allow-rename on
no server running on /root/.cache/xpanes/socket
+ exit 0
```

**Comment by greymd** [comment]
More simple one.

```
xpanes '' ''
```

![screen shot 2017-05-14 at 6 35 21 pm](https://cloud.githubusercontent.com/assets/3874767/26032687/200df096-38d4-11e7-81d4-689fb88acaa9.png)


And also it could be happened in both tmux session and normal session.

**Comment by greymd** [comment]
This is caused by the empty name of window. The solution would be like this. This change will most likely be applied to the `master` at version 2.1.0.

```diff
diff --git a/bin/xpanes b/bin/xpanes
index e7aa586..cde08d1 100755
--- a/bin/xpanes
+++ b/bin/xpanes
@@ -316,6 +316,9 @@ xpns_arr2args() {
 #   return   =  aaa-12345(PID)
 xpns_generate_window_name() {
   cat <<<"$1" \
+    | \
+    # Put "EMPTY" in case of empty string.
+    sed 's/^$/EMPTY/' \
     | tr '.' ' ' \
     | \
     # coron must be replaced because tmux interprets it.

```

**Comment by greymd** [comment]
Fixed with [v2.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v2.1.0).
